tests/dummy_plugin/dummy.py

```diff
@@ -6,4 +7,10 @@ class DummyTest(BotPlugin):
     """
     @botcmd
     def foo(self, msg, args):
+        """This runs foo."""
+        return 'bar'
+
+    @re_botcmd(pattern=ur"plz dont match this")
+    def re_foo(self, msg, match):
```

Comment:
yeah, just fixed that. Whatever insane mechanism backports python 3 -> 2 on `pip install` left me with that handy artifact.